### PR TITLE
[bugfix] Confirm mint completion before metadata update.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- remove polling logic for mint transaction.
- confirm mint completion before metadata update.

### CODE REVIEW

File: common/ainft.js [hunk 1]
```diff
@@ -7,17 +7,40 @@ const {
...
+const TOKEN_CHECK_INTERVAL_MS = 5 * 1000; // 5sec
+
 const setMetadata = async (objectId, tokenId, metadata) => {
   try {
     const object = await ainft.nft.get(objectId);
-    const token = await object.getToken(tokenId);
+    let token;
+
+    const startTime = Date.now();
+    while (!token && Date.now() - startTime < MAX_WAIT_TIME_MS) {
```
Comment (platfowner) [inline common/ainft.js:27]: could you add a note or some comments if this approach is experimental.

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: common/ainft.js (hunk 1 above)
Comment (jiyoung-an) [reply common/ainft.js:28]: done!

**Comment by jiyoung-an** [comment]
Thank you for quick review @platfowner! now merging...

**Pull request merged**